cmd/build-iso.go

```diff
@@ -140,8 +138,6 @@ func NewBuildISO(root *cobra.Command, addCheckRoot bool) *cobra.Command {
 	c.Flags().String("overlay-iso", "", "Path of the overlayed iso data")
 	c.Flags().String("label", "", "Label of the ISO volume")
 	c.Flags().Bool("bootloader-in-rootfs", false, "Fetch ISO bootloader binaries from the rootfs")
-	c.Flags().Var(firmType, "firmware", "Firmware to install for: 'efi' or 'bios'. (defaults to 'efi')")
```

Comment:
I wouldn't remove options but just make them output a warning.

Reply:
Done, now if a non supported firmware (basically anything but `efi`) is provided it will just error out with a non supported error message. If `--firmware efi` is provided then the deprecated warning appears but it does not error out. `--firmware` option, as it is deprecated it does not appear in help message.